### PR TITLE
Capacity Server (Grid) Improvements

### PR DESCRIPTION
**Pull request by maxux**
#### What this PR resolves:
- Improve InfluxDB support on the grid capacity
- Add a better way to starts the grid capacity server from scratch
- Add configuration command line tool

### CODE REVIEW

File: JumpscaleLib/servers/grid_capacity/server/influxdb/__init__.py [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
+_client = None
+influxlog = j.logger.get("grid-influx")
 
 def init(settings):
-    if not settings.INFLUX_HOST or \
-            not settings.INFLUX_PORT or \
-            not settings.INFLUX_DB:
-        return
+    if not settings.INFLUX_HOST:
+        return False
```
[the remark this answers is not in the document]
Comment (maxux) [reply JumpscaleLib/servers/grid_capacity/server/influxdb/__init__.py]: Code can still works even if influxdb is not set. False is just an indication but is not checked.

File: JumpscaleLib/servers/grid_capacity/CommandLine.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
+from jumpscale import j
```
[the remark this answers is not in the document]
Comment (maxux) [reply JumpscaleLib/servers/grid_capacity/CommandLine.py]: It looks weird to have command line on core, which depends on libs. That's why the file is here, waiting for a better place. Does core is the way to go ?

File: JumpscaleLib/servers/grid_capacity/CommandLine.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+import click
+
+logger = j.logger.get("capacity", force=True)
+j.logger.loggers_level_set(level='DEBUG')
+
+@click.group()
+def cli():
+    pass
+
+def die(message):
```
[the remark this answers is not in the document]
Comment (maxux) [reply JumpscaleLib/servers/grid_capacity/CommandLine.py]: This is just a simplification for print the error and close. This is not made for exception handling. It's just a nice ui trick.

File: JumpscaleLib/servers/grid_capacity/CommandLine.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (zaibon) [reply JumpscaleLib/servers/grid_capacity/CommandLine.py]: I'm not saying to put it in core. it must be here in lib. this was just a link to show how to do it

**Comment by maxux** [comment]
Everything should be implemented now :)

**Comment by zaibon** [comment]
@maxux you need to change the status of this PR so I can merge

**Pull request merged**